frontend/src/models/dashboardsModel.js

```diff
@@ -118,7 +118,10 @@ export const dashboardsModel = kea({
                 return [...list.filter((d) => d.pinned), ...list.filter((d) => !d.pinned)]
             },
         ],
-        dashboardsLoading: [() => [selectors.rawDashboardsLoading], (rawDashboardsLoading) => rawDashboardsLoading],
+        dashboardsLoading: [
+            () => [selectors.rawDashboardsLoading, selectors.sharedDashboardsLoading],
+            (a, b) => a || b,
```

Comment:
Are a and b the respective selector values? Can we make it even more obvious if so? 😅

Reply:
:D I thought they were already obvious this way, but can be verbose if needed.

Reply:
I think slight verbosity in kea files helps sometimes because of its somewhat strange syntax, at least for those new and unfamiliar to it

Reply:
I updated this already.

However I think this falls under https://posthog.com/handbook/engineering/how-we-review:

> What not to look for:
> "I would have done it differently" (Unless the code is completely incomprehensible or unreadable, or will cause us massive harm long term - as long as it works, it's good enough.)